networking/spoke-BU0001A0042.json

```diff
@@ -0,0 +1,256 @@
+{
```

Comment:
change the naming taking into account that it is going to be the same application in multiple regions `spoke-BU0001A0042-00` and `spoke-BU0001A0042-01`.

Initially it is ok use same application, same region but the identifier needs to be differ.